spec/spec_helper.rb

```diff
@@ -6,10 +6,10 @@
   unless ENV['APPVEYOR']
     require 'coveralls'
 
-    SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+    SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
```

Comment:
Follow up warnings, `/home/runner/work/td-logger-ruby/td-logger-ruby/spec/spec_helper.rb:9:in `<top (required)>': [DEPRECATION] ::[] is deprecated. Use ::new instead.`